test/lib/mayaUsd/render/mayaToHydra/cpp/testIsolateSelect.py

```diff
@@ -53,6 +54,14 @@ class TestIsolateSelect(mtohUtils.MayaHydraBaseTestCase):
 
     _requiredPlugins = ['mayaHydraCppTests', 'mayaHydraFlowViewportAPILocator']
 
+    imageVersion = None
+
+    @classmethod
```

Comment:
For the unit tests, there are basically 2 possible workarounds to keep the same images as much as possible.
1. modifyDefaultLightIntensityIfUsdGreaterOrEqualTo_24_11() / resetDefaultLightIntensityIfUsdGreaterOrEqualTo_24_11()  to modify/reset the default light intensity which fixes most of the problematic tests.
2. When 1. doesn't work I need to do another baseline image and I am using the imageVersion from assertsnapshot where the new version of the image is in a given folder whose name is imageVersion (when it's not None).

Reply:
For 1. : Could we change the modify/reset methods to be more generic/modifiable over time than them being fixed to a single version, since I presume these things could change again? i.e. have something like setDefaultLightIntensityByUsdVersion and resetDefaultLightIntensity, in which we could handle all the different USD versions 

For 2. : Could we move the imageVersion class attribute and setup to the MayaHydraBaseTestCase in mtohUtils.py? That way the usd version would always be set automatically, can be used as needed and the code will not have to be copy-pasted in each test